src/chart/Treemap.tsx

```diff
@@ -77,6 +77,7 @@ export const addToTreemapNodeIndex = (
 };
 
 const options: ChartOptions = {
+  barCategoryGap: '10%',
```

Comment:
treemap has this? lol

Reply:
It does not. We might need to split the state types later, for now I decided to hardcore the default and never use it.